Kinvey.Tests/UserIntegrationTests.cs

```diff
@@ -336,6 +337,7 @@ public void TestMIC_LoginWithMIC_NormalFlow()
         }
 
         [TestMethod]
+        [Obsolete("This test has been deprecated.")]
         public void TestMIC_LoginWithMIC_NormalFlow_ClientID()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?